Description: Fixes error when running a recent version of automake
Index: gltron-0.70final/nebu/scripting/Makefile.am
===================================================================
--- gltron-0.70final.orig/nebu/scripting/Makefile.am
+++ gltron-0.70final/nebu/scripting/Makefile.am
@@ -6,8 +6,6 @@ libscripting_a_SOURCES= \
 libscripting_a_LIBADD= \
 	$(top_builddir)/lua/src/liblua.a \
 	$(top_builddir)/lua/src/lib/liblualib.a
-	# @top_srcdir@/lua/src/liblua.a \
-	# @top_srcdir@/lua/src/lib/liblualib.a
 
 AM_CFLAGS = \
 	-I$(top_srcdir)/nebu/include/base \
